### DISCUSSION TITLE
finest resolution

### DISCUSSION THREAD

**Discussion by Tungway1990** [NVlabs/instant-ngp#558] [Q&A] [answered]
Hi, 

One quick question, from the paper table 1 `Only the hash
table size T and max. resolution Nmax need to be tuned to the use case`

I can find `log2_hashmap_size` for `T` and `base_resolution` for `Nmin` in `base.json` but no Finest resolution (`Nmax`)

May I know the corresponding parameter for Finest resolution?

Thanks.

**Comment by Tom94** [comment, marked as answer]
Our configs have an entry for `b` (called `per_level_scale`) instead of `Nmax`.

`b` is set programmatically in `testbed.cu`: it's derived from `Nmax` (called `desired_resolution` in the code) as per equation 3 of the paper.
